### ISSUE TITLE
Feature Request: Signing Up With Email Or Phone Number On Any Device

### ISSUE THREAD

**Issue by trymeouteh** [wireapp/wire#165]
This is a minor issue for many but for me and many of my friends when we wanted to sign up to Wire we didn't want to sign up with our phone numbers but when we installed the app the only way to sign up was to use a Phone Number so we had to sign up on the computer. Some people I know don't have computers and only have their phones.

I would be beneficial to be able to sign up with email or phone number on any device especially the ability to use email on mobile phones since many people like to use Wire since it is lot WhatsApp or Signal without using your phone number as your contact. When my friends signed up I told them they can use an email but they were confused when they tried to sign up since they required a phone number since they tried signing up in the mobile app.

I think by adding this will remove some confusion for new users.

**Comment by kristin-wire** [comment]
Hi there, 

Thanks for your request. This is something we're continuing to discuss internally, but no concrete details to share just yet. 
For now, the work around is to register on desktop at app.wire.com with email, then toggle to Log in with email on your mobile device.

**Comment by ganego** [comment]
What is there to discuss. You want to advertise with privacy and unless you have intentions to collect and sell phone numbers or metadata, there is no reason to not have it implemented. Stll thinking about why I got a verification SMS without Wire knowing my phone number btw: https://github.com/wireapp/wire-android/issues/1310  

This is pretty much one of the most important features for normal users you have over WA and you decide to not make it available.  
Tried to get a colleague to Wire some days ago telling him he can get it without phone number - guess my surprise when I showed him the app. Not good advertisement.
